website/src/components/header.js

```diff
@@ -37,15 +56,27 @@ const Header = props => (
         </div>
       </Spacings.Inline>
     </Link>
-    <a href="https://github.com/commercetools/merchant-center-application-kit">
-      <GitHubSvg />
-    </a>
+    <Spacings.Inline alignItems="center">
+      <a
+        css={css`
+          display: flex;
+        `}
+        href="https://github.com/commercetools/merchant-center-application-kit"
+      >
+        <GitHubSvg />
+      </a>
+      <MenuButton onClick={props.toggleMenu}>
+        <BurgerIcon isActive={props.isMenuOpen} />
```

Comment:
The "burger menu" is visible on small screens

![2019-04-29 14 25 30](https://user-images.githubusercontent.com/1110551/56895932-b7037900-6a8a-11e9-820b-ebde1cee8866.gif)

Reply:
those pink underlines look weird. Shouldn't they be more inline-block than block? 🤔

Reply:
That looks pretty nice :)

Reply:
![Screen Shot 2019-04-29 at 2 28 43 PM](https://user-images.githubusercontent.com/2425013/56896088-1eb9c400-6a8b-11e9-89ef-58c08366aebd.png)

Reply:
Yeah, don't worry about that. I still need to go through the visual things with design.

Reply:
Actually that was easy to fix https://github.com/commercetools/merchant-center-application-kit/pull/573/commits/7a080053bb4415e6d1a7d534c7aee1dd918759fa

Reply:
<img width="442" alt="image" src="https://user-images.githubusercontent.com/1110551/56896833-32662a00-6a8d-11e9-8394-f77a85ca2b52.png">